RevenueCatUI/CustomerCenter/Data/CustomerCenterData.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  File.swift
+//  
+//
+//  Created by Cesar de la Vega on 28/5/24.
+//
+
+import Foundation
+
+struct CustomerCenterData: Decodable {
+
+    let id: String
+    let paths: [HelpPath]
+    let title: LocalizedString
+    let supportEmail: String
+    let appearance: Appearance
+
+    enum HelpPathType: String, Decodable {
+        case missingPurchase = "MISSING_PURCHASE"
```

Comment:
Should this maybe be `missingEntitlement`? I'm assuming this is the case where the purchase has been made, e.g. money has been transferred, but nothing is received in return. In that case `missingPurchase` doesn't fully describe the situation imo.

Reply:
Talked with the backend folks and they said let's keep as `MISSING_PURCHASE` for now since it's internal, will bring it back up whenever there's a PR up for the endpoint

Reply:
Alright, thanks for checking! Maybe we can still think about the user-facing text, but not necessarily now.